index.html

```diff
@@ -263,9 +265,9 @@ <h2>Extensions to the {{Performance}} interface</h2>
       performance metric data from the <a>Performance Timeline</a>.</p>
       <pre class="idl">
       partial interface Performance {
-        PerformanceEntryList getEntries ();
-        PerformanceEntryList getEntriesByType (DOMString type);
-        PerformanceEntryList getEntriesByName (DOMString name, optional DOMString type);
+        PerformanceEntryList getEntries (optional boolean includeFrames = false);
```

Comment:
This and the subsequent calls are a boolean trap. I just ran into this in a test that said `getEntries(true)` and it was awfully confusing.

Should be `getEntries({includeFrames: false})` or so.

Reply:
Updated; I've added an options dict for `getEntries` -- and only for `getEntries`, at that point, `getEntriesByType`/`getEntriesByName` are kind of redundant, so there's no benefit to changing their signatures.